alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/ScheduledTask.java

```diff
@@ -42,7 +42,9 @@
 public abstract class ScheduledTask implements Runnable {
     public static final String FORMAT_PATTERN = "MM/dd/yyy hh:mm a";
     public static final String STOP_SCHEDULE_EXPRESSION = "";
+    //Spring Cron documentation  https://riptutorial.com/spring/example/21209/cron-expression
     public static final String EVERY_MINUTE_CRON_EXPRESSION = "0 0/1 * 1/1 * *";
+    public static final String ONCE_DAILY_CRON_EXPRESSION = "0 0 0 * * *";
```

Comment:
I realize this is just for the patch, so this may not be an issue, but would it be a problem for customers in other timezones?

Reply:
We are always using UTC timezone.  So it would run based on the UTC timezone.

Reply:
Also this is not for the patch, this PR is for the current version of Alert.